lib/upfluence/error_logger/sentry.rb

```diff
@@ -6,19 +6,53 @@ module Upfluence
   module ErrorLogger
     class Sentry
       EXCLUDED_ERRORS = (
-        ::Sentry::Configuration::IGNORE_DEFAULT + ['ActiveRecord::RecordNotFound']
+        ::Sentry::Configuration::IGNORE_DEFAULT + [
+          'ActiveRecord::RecordNotFound',
+          'ActiveRecord::ConcurrentMigrationError'
+        ]
       )
+
+      class << self
+        def err_name_lambda(level, name)
+          lambda do |err|
+            level if err.class.name == name
```

Comment:
Rubocop complains that this should instead use `instance_of?` but that means importing the classes we want to test against as far as I understand, which is a bit too heavy imo

Reply:
i think your approach is better, can i simply ask you:


```suggestion
            level if err.class.name.eql? name
```